### PR TITLE
Bug fix - KB-5825 (connections count)

### PR DESCRIPTION
**Pull request by venkykandagaddala**

### CODE REVIEW

File: project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html [hunk 1]
```diff
@@ -26,11 +26,11 @@
         <mat-card class="flex-1 remove-botton-radius">
             <mat-card-header>
                 <mat-card-title class="margin-l-0 f-medium">
-                    <h2 *ngIf="data && data.length">{{data.length}}
-                        <span *ngIf="data && data.length > 1 "> {{ 'network.connectionRequests' | translate }}</span>
-                        <span *ngIf="data && data.length === 1 "> {{ 'network.tabConnectionRequest' | translate }}</span>
+                    <h2 *ngIf="filterdData && filterdData.length">{{filterdData.length}}
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html:29]: please do optional check for filterData

File: project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html [hunk 2]
```diff
@@ -39,16 +39,18 @@ <h4 class="w-full text-center ws-mat-black60-text" *ngIf="data && data.length <=
                         <ws-app-careers-card class="w-fix" [discuss]="discuss"></ws-app-careers-card>
                     </div>
                 </ng-container> -->
-                <ng-container *ngIf="data && data.length">
+                <ng-container *ngIf="filterdData && filterdData.length">
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html:42]: same as above

File: project/ws/app/src/lib/routes/network-v2/routes/network-my-connection/network-my-connection.component.html [hunk 3]
```diff
@@ -23,16 +23,16 @@
         </div>
     </div>
     <div class="flex flex-1 mb-4">
-        <mat-card class="flex-1 remove-botton-radius min-height">
+        <mat-card class="flex-1 remove-botton-radius">
             <mat-card-header>
                 <mat-card-title class="w-full margin-l-0 f-medium">
-                    <h2 *ngIf="data && data.length && !enableSearchFeature && data.length > 1 ">{{data.length}}
+                    <h2 *ngIf="filterdData && filterdData.length && !enableSearchFeature && filterdData.length > 1 ">{{filterdData.length}}
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/network-v2/routes/network-my-connection/network-my-connection.component.html:29]: same as above , for all instance

File: project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html (hunk 1 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html:29]: **filterdData** is alreay defined as array

File: project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html (hunk 2 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/network-v2/routes/network-connection-requests/network-connection-requests.component.html:42]: **filterdData** is alreay defined as array. I think no need to do the optional check again.

File: project/ws/app/src/lib/routes/network-v2/routes/network-my-connection/network-my-connection.component.html (hunk 3 above)
Comment (venkykandagaddala) [reply project/ws/app/src/lib/routes/network-v2/routes/network-my-connection/network-my-connection.component.html:29]: **filterdData** is alreay defined as array. I think no need to do the optional check again.

**Pull request merged**